README.md

```diff
@@ -1822,6 +1822,7 @@ The following configuration is available in `disco_elasticache.ini` to configure
     port=6379
     parameter_group=default.redis2.8
     num_nodes=2
+    maintenance_window=sun:5:00-sun:09:00
```

Comment:
I think you can only pass maintenance window in UTC time. We would have to adjust the value accordingly to match with our desired time in EST.